crates/integration_test/src/utils/test_utils.rs

```diff
@@ -189,7 +198,10 @@ pub fn test_inside_container(
     // assume that the create command was successful. If it wasn't we can catch that error
     // in the start_container, as we can not start a non-created container anyways
     std::thread::sleep(std::time::Duration::from_millis(1000));
-    match start_container(&id, &bundle).unwrap().wait_with_output() {
+    match start_container(&id.to_string(), &bundle)
```

Comment:
Hey can we extract the id_str as a variable, as it seems to be used in quite a few places later as well

Reply:
I have also changed the `create_container` function to take `&str` instead of `&Uuid` so that all those functions are consistent.